nulib/dc-api-v2
# IIIF Manifest for Works endpoint in DC API V2

## What does this do?
Adds `?as=iiif` to the end of the Work API endpoint, which returns a IIIF 3.0 Manifest.

## How to test
1. Run the DC API v2 locally
2. In the browser, call a single Work endpoint - `works/1234`
3. Add `?as=iiif` to the end - `works/1234?as=iiif`
4. Verify you're seeing a IIIF 3.0 manifest
5. Validate here: https://presentation-validator.iiif.io/
